components/tabs/tabs.js

```diff
@@ -781,18 +794,13 @@ class Tabs extends LocalizeCoreElement(ArrowKeysMixin(SkeletonMixin(LitElement))
 		if (selectedTab) {
 			this.#updateSelectedTab(selectedTab);
 		}
+		this.#setAriaControls();
 
 		await this.updateComplete;
 
 		if (!this._initialized && this._tabs.length > 0) {
 			this._initialized = true;
 		}
-
-		if (selectedTab) {
-			// set corresponding panel to selected
-			const selectedPanel = this._getPanel(selectedTab.id);
-			if (selectedPanel) selectedPanel.selected = true;
-		}
```

Comment:
This is already being done in `#updateSelectedTab` so cleaning it up (I believe I missed it when fixing up a merge conflict)